config/default/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
why change this to `Always`?
This can result in failed deployments in certain air-gapped environment even if they had images pre-loaded on the nodes.

Reply:
I'm not sure why this got updated, or even why its needed. I've removed this patch entirely.